### ISSUE TITLE
TypeError for gitlog variable

### ISSUE THREAD

**Issue by abhishekph** [domharrington/node-gitlog#52]
I am novice user of nodejs and gitlog.
Nodejs server.js file:
```
const express = require('express');
const bodyParser = require('body-parser');
const path = require('path');
const fs = require('fs');
//const info about DB
const nano = require('nano')(`http://${username}:${userpass}@localhost:5984/${dbName}`);
const gitlog = require("gitlog");

const app = express();

app.set('view engine', 'ejs');
app.set('views', path.join(__dirname, 'views'));

app.use(bodyParser.json());
app.use(bodyParser.urlencoded({extended: false}));

const options = {
  repo: "./data/test/repo",
  fields: ["authorName"],
};

const commits = gitlog(options);
console.log(commits);
const routes = require('./routes/routes.js')(app, fs, nano, gitlog);

const server = app.listen('3000', function(){
  console.log('Server started listening on port %s', server.address().port);
});

```
Though I can't log the commit since issue about gitlog as below.

```
[nodemon] restarting due to changes...
[nodemon] starting `node server.js`
C:\Users\test\Downloads\gitstat\server.js:24
const commits = gitlog(options);
                ^

TypeError: gitlog is not a function
    at Object.<anonymous> (C:\Users\test\Downloads\gitstat\server.js:24:17)
    at Module._compile (internal/modules/cjs/loader.js:1156:30)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:1176:10)
    at Module.load (internal/modules/cjs/loader.js:1000:32)
    at Function.Module._load (internal/modules/cjs/loader.js:899:14)
    at Function.executeUserEntryPoint [as runMain] (internal/modules/run_main.js:74:12)
    at internal/main/run_main_module.js:18:47
```

**Comment by hipstersmoothie** [comment]
Oops! during my TS rewrite I accidently changed the exports! You should be able to use `gitlog` by importing `require("gitlog")`

**Comment by hipstersmoothie** [comment]
:rocket: Issue was released in [`v4.0.0`](https://github.com/domharrington/node-gitlog/releases/tag/v4.0.0) :rocket:
